fidesctl/src/fidesctl/cli/util_comands.py

```diff
@@ -0,0 +1,131 @@
+"""Contains all of the Utility-type CLI commands for Fidesctl."""
+import os
+
+import click
+import requests
+import toml
+
+from fidesctl.cli.options import yes_flag
+from fidesctl.cli.utils import (
+    handle_cli_response,
+    pretty_echo,
+)
+from fidesctl.core import api as _api
+from fidesctl.core.utils import echo_green, echo_red
+
+
+@click.command()
+@click.pass_context
+def init(ctx: click.Context) -> None:
+    """
+    Initialize a Fidesctl instance.
+    """
+
+    # Constants
+    dir_name = ".fides"
+    config_file_name = "fidesctl.toml"
+    config_path = f"{dir_name}/{config_file_name}"
+    config = ctx.obj["CONFIG"]
+
+    # List the values we want to exclude from the user-facing config
+    # These are values that are generated at config load time
+    excluded_values = {
+        "user": {"request_headers"},
+        "api": {"async_database_url", "sync_database_url"},
+    }
+
+    print("Initializing Fidesctl...\n")
+
+    # create the .fides dir if it doesn't exist
+    if not os.path.exists(dir_name):
+        os.mkdir(dir_name)
+        echo_green(f"Created a '{dir_name}' directory.\n")
+    else:
+        print(f"Directory '{dir_name}' already exists. Skipping...\n")
+
+    # create a config file if it doesn't exist
+    if not os.path.isfile(config_path):
+        config_docs_url = "https://ethyca.github.io/fides/installation/configuration/"
+        config_message = f"""Created a config file at '{config_path}'. To learn more, see:
+            {config_docs_url}\n"""
+        with open(config_path, "w") as config_file:
+            config_dict = config.dict(exclude=excluded_values)
+            toml.dump(config_dict, config_file)
+        echo_green(config_message)
+
+    else:
+        print(f"Configuration file already exists at '{config_path}'. Skipping...\n")
```

Comment:
Same as my above comment.
```suggestion
        print(f"Configuration file already exists at '{config_path}'.\n")
```